### DISCUSSION TITLE
Missing Adjust for TV setting.

### DISCUSSION THREAD

**Discussion by blck-b0x** [elementary/switchboard-plug-display#386] [Q&A] [answered]
Hello, I'm currently running ZorinOS and was considering switching to elementary, but when I tried the live demo, I noticed the same issue I always have with my TV, overscan. I looked in the settings but could not find the option I have on zorin that adjusts the display to fix that. I searched online and found that most TVs seem to have some way or other to disable overscan and that is the most common answer for this, but my TV doesn't have that option, and since elementary doesn't have this option in settings I don't think Imma be able to switch.
Is there any other way to fix this? Maybe elementary has the option somewhere else and I just didn't find it?

![image](https://github.com/elementary/switchboard/assets/109315036/ed9ee93b-8e86-43e8-8996-074abbc62d58)
(Image of the setting I'm referring to in zorin)
Thanks

**Comment by danirabbit** [comment, marked as answer]
Hey thanks for reporting this! I've opened https://github.com/elementary/switchboard-plug-display/issues/387 to track this feature request. Closing this discussion since it's a feature that needs to be implemented
